README.md

```diff
@@ -87,7 +87,7 @@ You can configure the `CodeGenProcessor` as any Java annotation processor, here
     <!-- Configure the execution of the compiler to execute the codegen processor -->
     <plugin>
       <artifactId>maven-compiler-plugin</artifactId>
-      <version>3.1</version>
+      <version>3.5</version> <!-- since 3.5 maven-compiler-plugin was updated https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html -->
```

Comment:
why is the comment necssary ?

Reply:
AnnotationProcesserPaths  is new option for resolve annotation classpath on both `maven` (since3.5) and `gradle` (5.0) then your example is out of date.
I noted it for anyone in old environment